e2e/cypress/integration/integrations/custom_slash_commands/custom_slash_commands_spec.js

```diff
@@ -0,0 +1,257 @@
+// ***************************************************************
+
+// Group: @integrations
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+import {addNewCommand} from './slash_commands_spec';
+
+describe('Slash commands', () => {
+    const trigger = 'my_trigger';
+    let user1;
+    let user2;
+    let team1;
+    let commandURL;
+
+    before(() => {
+        cy.requireWebhookServer();
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            team1 = team;
+
+            cy.apiGetChannelByName(team.name, 'town-square').then(({channel}) => {
+                commandURL = `${Cypress.env().webhookBaseUrl}/send_message_to_channel?channel_id=${channel.id}`;
+            });
+
+            cy.apiCreateUser().then(({user: otherUser}) => {
+                user2 = otherUser;
+                cy.apiAddUserToTeam(team.id, user2.id);
+            });
+        });
+    });
+
+    it('MM-T696 Can\'t delete other user\'s slash command', () => {
+        cy.apiAdminLogin(user1);
+
+        // # Open slash command page
+        cy.visit(`/${team1.name}/integrations/commands/installed`);
+
+        // # Create new Slash command
+        addNewCommand(team1, trigger, 'http://dot.com');
+
+        cy.visit('/admin_console/user_management/permissions/system_scheme');
+
+        // # Click manage slash commands checkbox and confirm
+        cy.findByTestId('all_users-integrations-manage_slash_commands-checkbox').then((el) => {
+            if (!el.hasClass('checked')) {
+                el.click();
+            }
+        });
+        cy.get('#saveSetting').click();
+        cy.waitUntil(() => cy.get('#saveSetting').then((el) => {
+            return el[0].innerText === 'Save';
+        }));
+
+        // # Login as another user
+        cy.apiLogin(user2);
+
+        // # Open slash command page
+        cy.visit(`/${team1.name}/integrations/commands/installed`);
+
+        // * Verify slash command exists
+        cy.contains(`/${trigger}`);
+
+        // * Verify that Edit and Delete options do not show up
+        cy.contains('Edit').should('not.exist');
+        cy.contains('Delete').should('not.exist');
+
+        // # Cleanup command
+        cy.apiAdminLogin(user1);
+        deleteCommand(team1, trigger);
+    });
+
+    it('MM-T697 Delete slash command', () => {
+        // # Open slash command page
+        cy.visit(`/${team1.name}/integrations/commands/installed`);
+
+        // # Create new Slash command
+        addNewCommand(team1, trigger, 'http://dot.com');
+
+        deleteCommand(team1, trigger);
+
+        // # Go back to home channel
+        cy.visit(`/${team1.name}/channels/town-square`);
+        cy.wait(TIMEOUTS.TWO_SEC);
+
+        // # Run slash command
+        cy.get('#post_textbox', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').clear().type(`/${trigger}{enter}`);
```

Comment:
This can be simplified with `cy.postMessage`

Reply:
Similar for other occurrences in this same file.

Reply:
I think you missed this cy.postMessage here.

Reply:
This is not `cy.postMessage` it is throwing an error and no message should appear. `cy.postMessage` is not just summiting a message it is also waiting to the post, so in this case it will not work.